docs/unified-test-documentation/dasharo-compatibility/358-suspend-mechanism-switching-S0ix-S3.md

```diff
@@ -0,0 +1,110 @@
+# Dasharo Compatibility: Suspend mechanism switching (S0ix/S3)
```

Comment:
@johanes2115 please add relevant entry point to `mkdocs` file. Without it, the test suite will remain hidden

Reply:
@sulewskiprzemyslaw I added the entrypoint to mkdocs.md file.